src/sidebar/components/ThreadList.js

```diff
@@ -204,8 +208,14 @@ function ThreadList({ threads }) {
             );
           }
 
-          // Check if effect was called with `visibleThreads` value which does
-          // not match most recently rendered `visibleThreads`.
```

Comment:
The old code _suggested_ that `visibleThreads` did not match the `visibleThreads` for the most recent render, but I wanted to be more sure of this.